Common/Data/UniverseSelection/Universe.cs

```diff
@@ -174,7 +174,7 @@ public IEnumerable<Symbol> PerformSelection(DateTime utcTime, BaseDataCollection
         public virtual Security CreateSecurity(Symbol symbol, IAlgorithm algorithm, MarketHoursDatabase marketHoursDatabase, SymbolPropertiesDatabase symbolPropertiesDatabase)
         {
             // by default invoke the create security method to handle security initialization
-            return SecurityManager.CreateSecurity(algorithm.Portfolio, algorithm.SubscriptionManager, marketHoursDatabase, symbolPropertiesDatabase,
+            return SecurityManager.CreateSecurity(algorithm.GetAvailableDataFeeds(), algorithm.Portfolio, algorithm.SubscriptionManager, marketHoursDatabase, symbolPropertiesDatabase,
```

Comment:
It seems strange that invoking `CreateSecurity` will automatically add subscriptions for all available quote types... wouldn't it make more sense to push this responsibility to the caller of this function.

This function would accept `QuoteType` and pass it along to whomever needs it, but it would be a shame if I only want `QuoteType.Trade` data and all of a sudden now I'm being bombarded with `QuoteType.Quote` data as well.